README.md

```diff
@@ -1,7 +1,6 @@
 # Datadog Continuous Testing for Bitrise
 
-<!-- TODO add link to marketplace after we publish the step -->
-<!-- [![Visual Studio Marketplace Version]()][1001]  -->
+![Bitrise](https://img.shields.io/bitrise/datadog-mobile-app-upload)
```

Comment:
![image](https://github.com/DataDog/synthetics-test-automation-bitrise-step-upload-application/assets/9317502/785f85de-92cb-4846-81db-3f770d51c0b0)

Reply:
Yeah i know it's still broken. I've been reading but still can't figure out what token they expect. I was considering to just ask them in the email thread soon 🤔

Reply:
Since https://shields.io/badges/bitrise takes an `appId`, I believe it will only give the build status of that app, which we already have with the other badge on the right.

Instead, we could go with the [github release badge](https://shields.io/badges/git-hub-release):

![GitHub Release](https://img.shields.io/github/v/release/DataDog/synthetics-test-automation-bitrise-step-run-tests)

```md
![GitHub Release](https://img.shields.io/github/v/release/DataDog/synthetics-test-automation-bitrise-step-run-tests)
```

I also created a PR on the github action: https://github.com/DataDog/synthetics-ci-github-action/pull/245

Reply:
Nice cheers this was the back-up thing i was considering, but i guess i've wasted enough time on the other thing that i should just go with this now 👍

Reply:
what do you think about this 
https://github.com/DataDog/synthetics-test-automation-bitrise-step-upload-application/pull/21/commits/6bf3365da14a388be26b1749e1b569de252cacda

Reply:
![image](https://github.com/DataDog/synthetics-test-automation-bitrise-step-upload-application/assets/25252536/ce04b446-acbf-42f1-be24-8e99780d6e3d)

Reply:
Perfection! 👌